ui/apps/platform/src/Containers/VulnMgmt/List/Cves/CveBulkActionDialogue.js

```diff
@@ -30,12 +32,15 @@ const findCVEField = (policySections) => {
     return { policySectionIdx, policyGroupIdx };
 };
 
-const CveBulkActionDialogue = ({ closeAction, bulkActionCveIds }) => {
+const CveBulkActionDialogue = ({ closeAction, bulkActionCveIds, cveType }) => {
```

Comment:
Can you additionally pass `data.cve` to the modal and use that instead of parsing the ID? If the ID format changes, this will break and we are back to the repair shop.

Reply:
Because the labyrinthine way we had to construct the data flow in order to allow for the "infinite drilldown" style of vuln management, we do not have the data at the top level--we only have the query and query vars, which we pass down to the generic VM table component to make its own GraphQL call.